karate-core/pom.xml

```diff
@@ -119,6 +120,12 @@
             <version>${junit5.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>net.masterthought</groupId>
```

Comment:
not in `karate-core` use `karate-demo` which already has this masterthought dependency

actually if we just do that one line change (see next comment) I'm ok not to have any tests. this is in my opinion a very simple change